tomcat-gcp-lib/pom.xml

```diff
@@ -30,6 +30,38 @@
     </dependency>
 
     <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-all</artifactId>
```

Comment:
Why do we need to include all of these transitive dependencies even though we do not directly use them?